integration_tests/tests/invoice-athena-tests.ts

```diff
@@ -37,45 +40,82 @@ describe("\nExecute athena query to retrieve invoice data and validate that it m
+    for (let i = 0; i < s3Response.length; i++) {
+      expect(s3Response[i].invoice_receipt_id).toEqual(
+        queryObjectsVal[i].invoice_receipt_id
+      );
+      expect(s3Response[i].vendor_name).toEqual(
+        queryObjectsVal[i].vendor_name
+      );
+      expect(s3Response[i].total.toFixed(4)).toEqual(
+        queryObjectsVal[i].total
+      );
+      expect(s3Response[i].invoice_receipt_date).toEqual(
+        queryObjectsVal[i].invoice_receipt_date
+      );
+      expect(s3Response[i].subtotal.toFixed(4)).toEqual(
+        queryObjectsVal[i].subtotal
+      );
+      expect(s3Response[i].due_date).toEqual(queryObjectsVal[i].due_date);
+      expect(s3Response[i].tax.toFixed(2)).toEqual(queryObjectsVal[i].tax);
+      expect(s3Response[i].tax_payer_id).toEqual(
+        queryObjectsVal[i].tax_payer_id
+      );
+      expect(s3Response[i].item_id.toString()).toEqual(queryObjectsVal[i].item_id);
+      expect(s3Response[i].item_description).toEqual(
+        queryObjectsVal[i].item_description
+      );
+      expect(s3Response[i].service_name).toEqual(
+        queryObjectsVal[i].service_name
+      );
+      expect(s3Response[i].unit_price.toFixed(4)).toEqual(
+        queryObjectsVal[i].unit_price
+      );
+      expect(s3Response[i].quantity.toString()).toEqual(queryObjectsVal[i].quantity);
+      expect(s3Response[i].price.toFixed(4)).toEqual(
+        queryObjectsVal[i].price);
+    }
   });
 
   test("retrieved view query results should matches with s3", async () => {
-    const s3Array = await s3GetObjectsToArray(testObject.bucket, folderPrefix);
-    const s3Objects = s3Array.map(
+    const s3Response = await s3GetObjectsToArray(testObject.bucket,folderPrefix);
+    const s3Objects = s3Response.map(
       (element: {
         vendor_name: string;
         service_name: string;
         price: number;
+        quantity:number
       }) => {
         return {
           vendor_name: element.vendor_name,
           service_name: element.service_name,
           price: element.price,
+          quantity:element.quantity
         };
       }
     );
 
     const queryString =
       'SELECT * FROM "btm_billing_curated" ORDER BY service_name ASC';
     const queryId = await startQueryExecutionCommand(databaseName, queryString);
-    const strFromQuery = await queryObject(queryId);
-    const queryObjects = strFromQuery.map(
-      (element: {
-        vendor_name: string;
-        service_name: string;
-        price: string;
-      }) => {
-        return {
-          vendor_name: element.vendor_name,
-          service_name: element.service_name,
-          price: element.price,
-        };
-      }
-    );
-    expect(s3Objects).toEqual(queryObjects);
+    const queryObjects:BillingCurated[] = await queryObject(queryId);
+    for (let i = 0; i < s3Response.length; i++) {
+    expect(s3Objects[i].vendor_name).toEqual(queryObjects[i].vendor_name);
```

Comment:
Has this gone through a linter? It's not even indented?

Reply:
fixed just ran npm run husky:install and its fixed now